src/browser.js

```diff
@@ -5,8 +5,8 @@ function apply(src, tar) {
 	tar.data = src.response;
 }
 
-export function send(method, uri, opts) {
-	return new Promise((res, rej) => {
+exports.send = function send(method, uri, opts) {
+	return new Promise(function (res, rej) {
```

Comment:
Ah! This is the only line that's needed... missed the arrow.

Can't accept the rest of the `exports.` changes

Reply:
That's right! I've just reverted the exports to how it was before, thanks for reviewing so quickly! 🤗

Reply:
No problem, nice catch!